CWE/localisation/nwo_decisions.csv

```diff
@@ -377,4 +377,8 @@ snowy_mountains_scheme_title;The Snowy Mountains Scheme;;;;;;;;;;X
 snowy_mountains_scheme_desc;The Snowy Mountains Scheme is a hydroelectricity and irrigation complex in south-east Australia. It consists of sixteen major dams, seven power stations, a pumping station, and 225 kilometres of tunnels, pipelines and aqueducts and was constructed between 1949 and 1974. The Chief engineer was Sir William Hudson. It is the largest engineering project undertaken in Australia.;;;;;;;;;;X
 xea_enlarge_union_gp_title;Eurasian Union a Community of Equal Nations;;;;;;;;;;X
 xea_enlarge_union_gp;Eurasian Union difficult compromises;;;;;;;;;;X
-xea_enlarge_union_gp_desc;To join the Eurasian Union, a state needs to fulfil economic and political conditions, which require the consent of all existing members and the candidate's adoption of existing EEU laws and implementing previous decisions made by the Eurasian Economic Commission.;;;;;;;;;;X
\ No newline at end of file
+xea_enlarge_union_gp_desc;To join the Eurasian Union, a state needs to fulfil economic and political conditions, which require the consent of all existing members and the candidate's adoption of existing EEU laws and implementing previous decisions made by the Eurasian Economic Commission.;;;;;;;;;;X
+enel_title;Law 1643;;;;;;;;;;X
```

Comment:
For decision titles please use this style: Enact/ Establish X etc

So enel_title;Law 1643 should be "Enact Law 1643"

Same for iri_prodi_title;IRI Restructuring should be "Restructure the IRI"